### PR TITLE
[ADMIN] Export Timeline entities with translations & relations

### PR DESCRIPTION
**Pull request by Remg**

### CODE REVIEW

File: src/Admin/TimelineProfileAdmin.php [hunk 1]
```diff
@@ -70,4 +70,14 @@ protected function configureListFields(ListMapper $listMapper)
             ])
         ;
     }
+
+    public function getExportFields()
+    {
+        return [
+            'ID' => 'id',
+            'Titre (FR,EN)' => 'exportTitles',
```
Comment (thomasbisignani) [inline src/Admin/TimelineProfileAdmin.php]: missing white space here ? `(FR,EN)` > `(FR ,EN)`

File: src/Admin/TimelineThemeAdmin.php [hunk 2]
```diff
@@ -143,4 +143,15 @@ protected function configureListFields(ListMapper $listMapper)
             ])
         ;
     }
+
+    public function getExportFields()
+    {
+        return [
+            'ID' => 'id',
+            'Titre (FR,EN)' => 'exportTitles',
```
Comment (thomasbisignani) [inline src/Admin/TimelineThemeAdmin.php]: same here for white space

File: src/Admin/TimelineMeasureAdmin.php [hunk 3]
```diff
@@ -154,4 +154,17 @@ protected function configureListFields(ListMapper $listMapper)
             ])
         ;
     }
+
+    public function getExportFields()
+    {
+        return [
+            'ID' => 'id',
+            'Titre (FR,EN)' => 'exportTitles',
```
Comment (thomasbisignani) [inline src/Admin/TimelineMeasureAdmin.php]: same here for white space

File: src/Admin/TimelineProfileAdmin.php (hunk 1 above)
Comment (Remg) [reply src/Admin/TimelineProfileAdmin.php]: ok go for `(FR, EN)` 👌

**Comment by thomasbisignani** [comment]
:ok_hand:

**Pull request merged**